pkg/operator/sync.go

```diff
@@ -45,7 +44,7 @@ func (optr *Operator) syncCustomResourceDefinitions(config render.OperatorConfig
 	}
 
 	for _, crd := range crds {
-		crdBytes, err := render.PopulateTemplate(&config, ownedManifestsDir+crd)
+		crdBytes, err := PopulateTemplate(&config, ownedManifestsDir+crd)
```

Comment:
Use https://golang.org/pkg/path/#Join for constructing the Path. Assuming I'm reading this correctly. I saw `...Dir + crd`. And assumed Dir was as directory.